dispatches/case_studies/fossil_case/ultra_supercritical_plant/storage/double_loop/run_prescient_double_loop_usc.py

```diff
@@ -0,0 +1,90 @@
+from prescient.simulator import Prescient
+
+#point to rts_gmlc scenario data
+print("                      ")
+print("------Start Prescient Simulation-------")
+print("                      ")
+# raise Exception()
```

Comment:
THis file is missing a header hence the error we are seeing about 3 files missing headers (I think)